Stats/Supporting Files/en_gb.lproj/Localizable.strings

```diff
@@ -0,0 +1,451 @@
+//
+//  Localizable.strings
+//  Stats
+//
+//  Translated by Coopydood
+//
```

Comment:
you removed these lines from the header:
//  Using Swift 5.0.
//  Running on macOS 10.15.

and now the line number is not correct